Classes/Public/TextKit/AztecAttachmentManager.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 
 /// Wrangles attachment layout and exclusion paths for the specified UITextView.
 ///
-@objc public class AztecAttachmentManager : NSObject
```

Comment:
I'm trying to remember why I had added `@objc` and extended `NSObject`. I borrowed this class from WPTextKit and then did some tweaks so maybe it was something I missed. Or maybe I was just thinking we'd also have objc support.  
Not necessary for now regardless so 👍